docs/site/BelongsTo-relation.md

```diff
@@ -6,6 +6,8 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/BelongsTo-relation.html
 ---
 
+{% include note.html content="There are certain limitations to `inclusionResolver()`. See [Limitations](Relations#limitations)." %}
```

Comment:
There are limitation about the inclusion at the bottom of the page. Maybe we can add on that instead? Similarly for other pages. 
Also, I cannot see what's `Relations#limitations` referencing (sorry if i missed it!), could you please point it to me? thanks.

Reply:
Apologies, that's probably [a typo that was introduced](https://github.com/strongloop/loopback-next/pull/4448#discussion_r367913553) and should be pointing to [`Relations.md#limitations`](https://github.com/strongloop/loopback-next/pull/4448/files#diff-9ed0bbe458d37bc6e2b9cfbd734d5d2a) instead.

The warning alert is duplicated across `hasOne` `hasMany` and `belongsTo`. Maybe we could take this opportunity to consolidate them to the `Relations` page?